src/walletConnect/reducer.ts

```diff
@@ -87,7 +87,7 @@ export const reducer = (
             .includes(pendingAction.topic)
         ),
         pendingSessions: state.pendingSessions.filter(
-          (pendingSession) => pendingSession.params.expiry > action.dateInSeconds
+          (pendingSession) => pendingSession.params.expiryTimestamp > action.dateInSeconds
```

Comment:
@MuckT codecov is complaining about this line not having tests. Maybe worth adding one given this is a more complex reducer?
Also, would this need a migration? could there be a persisted pendingSession with `expiry` instead of `expiryTimestamp`?

Reply:
I thought pending sessions where pretty ephemeral, but at second glance we might need migration otherwise the old ones may never expire.

Reply:
Cleared pending sessions with a migration in 057244a9b0e71725f532bed91d53e953ee230cf7 and tested in 04a4fbcb506947f05238199673b9f969df58e103.

Reply:
If the only change in schema is expiry to expiryTimestamp, then an inplace migration is fine, but if other fields are also changing, clearing seems good. Also, just for my understanding, how do pending sessions work when the app restarts?